google-cloud-tools-plugin/src/com/google/gct/idea/debugger/CloudDebuggerClient.java

```diff
@@ -46,8 +46,8 @@
  */
 public class CloudDebuggerClient {
 
-  private static final int LONG_CONNECTION_TIMEOUT_MS = 120 * 1000;
-  private static final int SHORT_CONNECTION_TIMEOUT_MS = 10 * 1000;
+  static final int LONG_CONNECTION_TIMEOUT_MS = 120 * 1000;
```

Comment:
I see what you are saying. If I'd correctly followed what I just said above, it should have been made public. However it is not currently used anywhere, so I'll add the annotation to avoid confusion, and it can be removed later if need be.